lib/propolis/src/hw/ps2ctrl.rs

```diff
@@ -269,16 +505,19 @@ impl PS2Ctrl {
         // We currently choose to mimic qemu, which gates the keyboard interrupt
         // with the keyboard-clock-disable in addition to the interrupt enable.
         let pri_pin = state.pri_pin.as_ref().unwrap();
-        pri_pin.set_state(
-            state.ctrl_cfg.contains(CtrlCfg::PRI_INTR_EN)
-                && !state.ctrl_cfg.contains(CtrlCfg::PRI_CLOCK_DIS)
-                && state.pri_port.has_output(),
-        );
+        if state.ctrl_cfg.contains(CtrlCfg::PRI_INTR_EN)
```

Comment:
Now that this is no longer updating the level of the interrupt pin, but rather pulsing it events, maybe change the function name to better reflect its usage?

Reply:
we chatted about this offline -- `update` might be misleading, but I can't think of a better name either. punting for now